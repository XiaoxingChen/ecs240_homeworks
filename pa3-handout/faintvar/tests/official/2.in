c Straightline code
p 3 5 4 1 5
c bb1: ;
b 1
e 1 2
c bb2: v1 = c;
b 2 1
e 2 3
c bb3: v2 = c;
b 3 2
e 3 4
c bb4: v3 = c;
b 4 3
e 4 5
c bb5: print(v1, v2, v3);
b 5 0 1 2 3
