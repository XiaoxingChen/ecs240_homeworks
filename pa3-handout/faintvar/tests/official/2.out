fvin 1 1 2 3
fvin 2 1 2 3
fvin 3 2 3
fvin 4 3
fvin 5
